src/command_line_parser.cc

```diff
@@ -461,6 +462,11 @@ TritonParser::SetupOptions()
       {OPTION_MODEL_NAMESPACING, "model-namespacing", Option::ArgBool,
        "Whether model namespacing is enable or not. If true, models with the "
        "same name can be served if they are in different namespace."});
+  model_repo_options_.push_back(
+      {OPTION_PEER_ACCESS, "enable-peer-access", Option::ArgBool,
+       "Whether GPU peer access is enable or not. If true, GPUs can share "
+       "virtual memory address,"
+       "adds cuda context "});
```

Comment:
```suggestion
       "Whether the server tries to enable peer access or not. Even when this options is set to true,  "
       "peer access could still be not enabled because the underlying system doesn't support it."
       " The server will log a warning in this case. Default is true."});
```